nextflow_schema.json

```diff
@@ -0,0 +1,192 @@
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row.",
+                    "fa_icon": "fas fa-file-csv"
+                },
+                "fastq_dir": {
+                    "type": "string",
+                    "fa_icon": "fas fa-folder-open",
+                    "description": "Path to directory containing paired (_R1/_R2) gzipped fastq files",
+                    "format": "directory-path"
+                }
+            }
+        },
+        "filtering_options": {
+            "title": "Filtering options",
+            "type": "object",
+            "description": "Options on minimum values for filtering out input data",
+            "default": "",
+            "properties": {
+                "kraken2_db": {
+                    "type": "string",
+                    "default": "s3://genome-idx/kraken/standard_08gb_20240904",
+                    "fa_icon": "fas fa-server",
+                    "description": "Kraken2 database to use for abundance checks"
+                },
+                "min_abundance_percent": {
+                    "type": "number",
+                    "default": 10,
+                    "description": "Minimum L.pn abundance to keep sample in pipeline",
+                    "fa_icon": "fas fa-percentage",
+                    "minimum": 1,
+                    "maximum": 99
+                },
+                "min_reads": {
+                    "type": "integer",
+                    "default": 150000,
+                    "description": "Minimum number of paired reads to keep sample in pipeline",
+                    "minimum": 0
+                }
+            },
+            "fa_icon": "fas fa-filter"
+        },
+        "skipping_options": {
+            "title": "Skipping options",
+            "type": "object",
+            "description": "Processes that can be skipped through parameters",
+            "default": "",
+            "properties": {
+                "skip_el_gato": {
+                    "type": "boolean",
+                    "description": "Skip running all el_gato Sequence Typing aspects",
+                    "fa_icon": "fas fa-forward"
+                },
+                "skip_plotting": {
+                    "type": "boolean",
+                    "description": "Skip plotting el_gato allele profiles",
+                    "fa_icon": "fas fa-forward"
+                }
+            },
+            "fa_icon": "fas fa-forward"
+        },
+        "quast_options": {
+            "title": "Quast options",
+            "type": "object",
+            "description": "Options for quast",
+            "default": "",
+            "properties": {
+                "quast_ref": {
+                    "type": "string",
+                    "default": "data/C9_S.reference.fna",
```

Comment:
Is a reference for quast in your pipeline strictly required?

For integration with IRIDA Next, I'd recommend marking this as `"hidden": true` so it doesn't show up in the IRIDA Next UI if it's not required.